src/main/resources/jenkins/branch/OrganizationFolderEmptyView/main.jelly

```diff
@@ -22,10 +22,26 @@
  ~ THE SOFTWARE.
  -->
 <?jelly escape-by-default='true'?>
-<st:compress xmlns:st="jelly:stapler">
-  <p>
-    This project scans branches in your SCM and generate a job for each of them,
-    but you have no branches configured.
-    <a href="./configure">Configure the project</a> and then <a href="./indexing">index branches</a> to get started.
-  </p>
-</st:compress>
+<div xmlns:j="jelly:core" xmlns:st="jelly:stapler" xmlns:f="/lib/form">
+    <div style="margin: 20px 0; padding: 10px 30px 20px 30px; background: #f8f8f8; border-radius: 10px; max-width: 55em;">
```

Comment:
[2 space indent in XML](https://github.com/jenkinsci/branch-api-plugin/blob/master/CONTRIBUTING.md#indentation)